DatadogSessionReplay/Tests/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UIImageViewWireframesBuilderTests.swift

```diff
@@ -0,0 +1,94 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import XCTest
+@testable import DatadogSessionReplay
+
+class UIImageViewWireframesBuilderTests: XCTestCase {
```

Comment:
Coverage here could be increased. Maybe we could discuss the plan for testing more elaborate wireframes builders.

Reply:
I like that. I'll keep these until we have snapshot in place. I'll describe the work in the ticket.